docs/_Research-Data-Management/20-pids.md

```diff
@@ -20,19 +20,19 @@ For example, Mathias Mueller is a fairly common name in Germany. However, with a
 ## DOI
 Digital Object Identifier or DOI is another form of persistent identifier. Papers, articles, and published datasets may have a DOI which links these items. This makes it easier to find these digital objects online. Even if the publisher changes the DOI of the published article, dataset, etc. will not change.
 
-
 # Benefits of assigning PIDs
 + Increase visibility
 + Connects the author with the work
 + You can use ORCID to login to other applications (ie. Coscine)
-
-# Disambiguation
-
-# Syntax
++ Machine readable
++ Adheres to FAIR principle
++ Permanent link to digital object
 
 # Use cases
+[doi: 10.1016/j.patter.2020.100180](https://doi.org/10.1016/j.patter.2020.100180)
```

Comment:
thanks for these edits, could you please put the reference in a context, the doi is a bit lonely here.

I would be even better to add the doi to the https://github.com/NFDI4Microbiota/nfdi4microbiota-knowledge-base/blob/main/docs/_bibliography/references.bib file.